src/pid/mountinfo.rs

```diff
@@ -0,0 +1,178 @@
+//! Information about mounts from `/proc/[pid]/mountinfo`.
+
+use std::fs::File;
+use std::io::{BufRead, BufReader, Result};
+use std::path::PathBuf;
+use std::str::{self, FromStr};
+
+use libc::pid_t;
+
+use parsers::{map_result, parse_isize, parse_usize};
+
+/// Process mounts information.
+///
+/// See `man 5 proc` for format details.
+#[derive(Debug, PartialEq, Eq, Hash)]
+pub struct Mountinfo {
+    /// Unique ID for the mount.
+    pub mount_id: isize,
+    /// ID of the parent mount.
+    pub parent_id: isize,
+    /// Device major ID (class).
+    pub major: usize,
+    /// Device minor ID (instance).
+    pub minor: usize,
+    /// Pathname which forms the root of this mount.
+    pub root: PathBuf,
+    /// Mount pathname relative to the process's root.
+    pub mount_point: PathBuf,
+    /// mount options.
+    pub mount_options: Vec<String>,
```

Comment:
I have a couple of doubts on this:
 * looking at kernel code, I don't see the `Other(String)` case
 * this exposes some opaque internal kernel properties and couple the library to specific/current kernel versions